ext/storage_sources/dir_store/dir_store.c

```diff
@@ -77,6 +79,7 @@ typedef struct {
      */
     uint32_t delay_ms;    /* Average length of delay when simulated */
     uint32_t force_delay; /* Force a simulated network delay every N operations */
+    uint32_t error_ms;    /* Average length of sleep when simulated */
```

Comment:
Alphabetize this in the list of items.

Reply:
Fixed in 0e6b43406c158379072274241f2c15de26c317b5